charms/build-and-release-easyrsa.sh

```diff
@@ -48,6 +44,6 @@ touch report.xml
 if [ ${RELEASE} = true ]; then
   CHARM=$(/usr/bin/charm push $JUJU_REPOSITORY/builds/easyrsa cs:~containers/easyrsa | head -n 1 | awk '{print $2}')
   echo "Releasing ${CHARM}"
-  charm release ${CHARM} --channel ${RELEASE_TO_CHANNEL} -r easyrsa-${RESOURCE_REV}
+  charm release ${CHARM} --channel ${RELEASE_TO_CHANNEL} -r easyrsa-5
```

Comment:
Yeah, this is a short-term step back but I think it's a move in the right direction.

We currently surface the RESOURCE_REV as a job parameter in Jenkins, meaning the user has to specify it when running a job. That's error-prone, especially when we're building all the charms as part of a single job. We've had problems with messed up revision numbers in a few of our releases already.

Hard-coding has obvious problems of its own, as you're right to point out. Updating resources will require a commit to bump the revs. We don't want that kind of friction in our release process.

I made this change to localize the badness, if that makes sense. Passing an error-prone parameter through three scripts and a GUI just doesn't feel right.

What we really need is to have our charm build scripts always grab the latest revisions, or alternatively, build and push their own resources along with the charm.